.github/workflows/reusable-macos.yml

```
@@ -47,11 +47,23 @@ jobs:
           ${{ inputs.free-threading && '--disable-gil' || '' }} \
           --prefix=/opt/python-dev \
           --with-openssl="$(brew --prefix openssl@3.0)"
+    - name: Build CPython for compiler warning check
+      if : ${{ !inputs.free-threading && inputs.os == 'macos-13 }}
```

The CI didn't start:

> Invalid workflow file: .github/workflows/build.yml#L174
The workflow is not valid. In .github/workflows/build.yml (Line: 174, Col: 11): Error from called workflow python/cpython/.github/workflows/reusable-macos.yml@d7b53dbc48f0ec2597990fd7ff1677f5c6ab25e1 (Line: 51, Col: 12): The expression is not closed. An unescaped ${{ sequence was found, but the closing }} sequence was not found.


https://github.com/python/cpython/actions/runs/10408250107?pr=123020

Here we go:

```suggestion
      if : ${{ !inputs.free-threading && inputs.os == 'macos-13' }}
```